### PR TITLE
DAOS-9868 test: collect more ofi logs

### PR DESCRIPTION
**Pull request by wangshilong**
Test-tag: daos_test
Signed-off-by: Wang Shilong <shilong.wang@intel.com>

### CODE REVIEW

**Comment by wangshilong** [comment]
> Is this meant to be permanent? Do we know how much large this makes the logs?

there should be very fewer logs if everything goes fine, and if there are some more logs, that is expected for debugging purpose.

File: src/tests/ftest/daos_test/suite.yaml [hunk 1]
```diff
@@ -59,6 +59,7 @@ server_config:
         - DD_MASK=mgmt,io,md,epc,rebuild,any
         - D_LOG_FILE_APPEND_PID=1
         - D_LOG_FLUSH=DEBUG
+        - FI_LOG_LEVEL=warn
```
Comment (bfaccini) [inline src/tests/ftest/daos_test/suite.yaml:62]: Don't you want to also set D_LOG_STDERR_IN_LOG=1 in order to merge libfabric debug logs in engine's log ??

Comment (wangshilong) [reply src/tests/ftest/daos_test/suite.yaml:62]: Yup, this should be included too, i was not aware of this, thanks!

**Review by daltonbohning — APPROVED** [review]

**Review by bfaccini — APPROVED** [review]

**Comment by wangshilong** [comment]
ping @daos-stack/daos-gatekeeper ...

**Pull request merged**